package-rust.py

```diff
@@ -176,15 +180,26 @@ def run(args):
 if channel == "stable":
     CFG_RELEASE=CFG_RELEASE_NUM
     CFG_PACKAGE_VERS=CFG_RELEASE_NUM
+    # UpgradeCode shoud stay the same for all MSI versions in channel
+    CFG_UPGRADE_CODE="1C7CADA5-D117-43F8-A356-DF15F9FBEFF6"
```

Comment:
Yes, they'd all end up on the PATH and then the first one wins.   In order to support side-by-side installation of multiple Rust versions, we should probably consider leaving PATH alone (or at least making this feature not selected by default), and providing shortcuts in the Start menu that open console window with PATH set for the appropriate Rust version.

Another interesting question is whether 32-bit and 64-bit versions should be considered the same product.   Right now they are, and installing, say, a 64-bit version uninstalls the 32-bit one (and vice-versa), which is kinda bad.
On the other hand, if we bundled both 32- and 64-bit versions of Rust libraries, there'd be no reason to want to install both, because one could just use the `--target` switch to select bitness of the generated executables.